server/middleware/cmd2faSessionExpiry.ts

```diff
@@ -13,7 +13,7 @@ export default class CmdSessionExpiry {
   ) {
     try {
       if (hmppsAuthMFAUser) return next()
-      const userSessionExpiryDateTime = await this.userAuthenticationService.getSessionExpiryDateTime(username)
+      const userSessionExpiryDateTime = (await this.userAuthenticationService.getSessionExpiryDateTime(username)) || []
```

Comment:
needed this because test user bobby brown not in the database, but seems more robust generally